styles/styles.css

```diff
@@ -604,6 +605,13 @@ main .section.bold-headings h6 {
   text-align: center;
 }
 
+/* section - full-width */
```

Comment:
There are already same styles defined below. Why do we need to repeat?

Reply:
Sure. Fixed